Tools/MiniBrowser/playstation/CMakeLists.txt

```diff
@@ -27,9 +27,10 @@ WEBKIT_EXECUTABLE(MiniBrowser)
 
 target_link_options(MiniBrowser PRIVATE -Wl,--no-demangle)
 
-#
-# Set the debugger working directory for Visual Studio
-#
 if (${CMAKE_GENERATOR} MATCHES "Visual Studio")
+    # Set the debugger working directory for Visual Studio
     set_target_properties(MiniBrowser PROPERTIES VS_DEBUGGER_WORKING_DIRECTORY "${CMAKE_RUNTIME_OUTPUT_DIRECTORY}")
+
+    # Set the startup target to MiniBrowser
+    set_property(DIRECTORY ${PROJECT_SOURCE_DIR} PROPERTY VS_STARTUP_PROJECT MiniBrowser)
```

Comment:
What happens when ENABLE_WEBCORE is ON and both framework, JSC and MiniBrowser, set the property? Does later wins simply?

Reply:
https://github.com/WebKit/WebKit/pull/5718/files#r1003839868 answers your question